### PR TITLE
Update local testing page to address upgrade to GameLift SDK v5

### PR DESCRIPTION
**Pull request by AMZN-daimini**
Also fixes a small typo.

### CODE REVIEW

**Review by cgalvan — APPROVED** [review]

File: content/docs/user-guide/gems/reference/aws/aws-gamelift/build-packaging-for-windows.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ Creating a dedicated server package includes the following steps:
 3.  Test the dedicated server package on your local machine.
 
 {{< note >}}  
-Debug dedicated servers are not supported as they require the debug visual studio redistributables which cannot be resdistributed. See [Distributable Code for Visual Studio 2019](https://docs.microsoft.com/en-us/visualstudio/releases/2019/redistribution) for more details.
+Debug dedicated servers are not supported as they require the debug visual studio redistributables which cannot be redistributed. See [Distributable Code for Visual Studio 2019](https://docs.microsoft.com/en-us/visualstudio/releases/2019/redistribution) for more details.
```
Comment (AMZN-Gene) [inline content/docs/user-guide/gems/reference/aws/aws-gamelift/build-packaging-for-windows.md:17]: Caps "Visual Studio"

**Review by AMZN-Gene — APPROVED** [review]

File: content/docs/user-guide/gems/reference/aws/aws-gamelift/local-testing.md [hunk 2]
```diff
@@ -1,41 +1,78 @@
...
+
 
-```shell
+## 2. Create an Anywhere Fleet
 
-$ java -jar GameLiftLocal.jar -p 9080
+Creating an Anywhere fleet is a much faster process compared to creating a regular AWS fleet, which usually takes about an hour to setup.
 
+```sh
+aws gamelift create-fleet --name AnywhereFleet --compute-type ANYWHERE --locations Location=custom-location-1 --region <Region>
```
Comment (AMZN-Gene) [inline content/docs/user-guide/gems/reference/aws/aws-gamelift/local-testing.md:35]: Add a note to save the Fleet-Id for the next steps (maybe provide an example of what a fleet id looks like)

**Review by AMZN-Gene — APPROVED** [review]

File: content/docs/user-guide/gems/reference/aws/aws-gamelift/local-testing.md [hunk 3]
```diff
@@ -1,41 +1,78 @@
...
+You will also need to fill in the server parameters via the following CVARs:
+- `sv_gameliftAnywhereWebSocketUrl`
+- `sv_gameliftAnywhereAuthToken`
+- `sv_gameliftAnywhereFleetId`
+- `sv_gameliftAnywhereHostId`
+- `sv_gameliftAnywhereProcessId`
 
-You can set this value by creating a settings registry file named `commands.MyProject_serverlauncher.setreg` in the `MyProject/Registry` folder with the following contents:
+All of the values for these properties are retrieved in the previous steps, aside from these notes:
+- In the `HostId` property should be filled with the `ComputeName`;
```
Comment (willihay) [inline content/docs/user-guide/gems/reference/aws/aws-gamelift/local-testing.md:67]:
```suggestion
- The `HostId` property should be set to the `ComputeName`.
```

**Review by willihay — APPROVED** [review]

File: content/docs/user-guide/gems/reference/aws/aws-gamelift/build-packaging-for-windows.md (hunk 1 above)
Comment (chanmosq) [reply content/docs/user-guide/gems/reference/aws/aws-gamelift/build-packaging-for-windows.md:17]:
```suggestion
Debug dedicated servers are not supported as they require the debug Visual Studio redistributables which cannot be redistributed. See [Distributable Code for Visual Studio 2019](https://docs.microsoft.com/en-us/visualstudio/releases/2019/redistribution) for more details.
```

**Pull request merged**